examples/pytorch/moco.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from torch import nn
+import torchvision
+import copy
+
+import lightly
+from lightly.models.modules import MoCoProjectionHead
+from lightly.models.utils import deactivate_requires_grad
+from lightly.models.utils import update_momentum
+
+
+class MoCo(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+
+        self.backbone = backbone
+        self.projection_head = MoCoProjectionHead(512, 512, 128)
+
+        self.backbone_momentum = copy.deepcopy(self.backbone)
+        self.projection_head_momentum = copy.deepcopy(self.projection_head)
+
+        deactivate_requires_grad(self.backbone_momentum)
+        deactivate_requires_grad(self.projection_head_momentum)
+
+    def forward(self, x_q, x_k):
```

Comment:
same argument here as for BYOL: wouldn't it be better to split the forward passes for the encoder and the momentum encoder?